### PR TITLE
SQL: tests metadata fields

### PR DESCRIPTION
**Pull request by unrealwork**
## SQL: tests metadata fields

- propertyUrl tests
- column datatype tests

### Review tasks

### CODE REVIEW

**Comment by KutuzovGregory** [comment]
ok

**Comment by Sild** [comment]
ok

**Comment by rmakulov** [comment]
ok

File: src/test/java/com/axibase/tsd/api/method/sql/metadata/SqlMetaDataTests.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        resultTable = executeQuery(sqlQuery).readEntity(StringTable.class);
+    }
+
+
+    /**
+     * Following tests is related to bug #2973
+     */
+
+    /**
+     * redmine: #2973
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/metadata/SqlMetaDataTests.java]: please leave only number

File: src/test/java/com/axibase/tsd/api/method/sql/metadata/SqlMetaDataTests.java [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+                "WHERE entity = 'sql-metadata-entity'";
+        sendSamplesToSeries(testSeries,
+                new Sample("2016-06-29T08:00:00.000Z", "0.05")
+        );
+        resultTable = executeQuery(sqlQuery).readEntity(StringTable.class);
+    }
+
+
+    /**
+     * Following tests is related to bug #2973
```
Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/sql/metadata/SqlMetaDataTests.java]: anywhere numbers are specified above each tests

**Pull request merged**